app/stac_api/serializers.py

```diff
@@ -183,3 +192,106 @@ def update(self, instance, validated_data):
 
         instance.save()
         return instance
+
+
+class ItemLinkSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = ItemLink
+        fields = ['href', 'rel', 'link_type', 'title']
+
+
+class ItemsPropertiesSerializer(serializers.Serializer):
+    # pylint: disable=abstract-method
+    # ItemsPropertiesSerializer is a nested serializer and don't directly create/write instances
+    # therefore we don't need to implement the super method create() and update()
+    datetime = serializers.DateTimeField(required=True, source='properties_datetime')
+    eo_gsd = serializers.ListField(required=True, source='properties_eo_gsd')
+    title = serializers.CharField(required=True, source='properties_title', max_length=255)
+
+    def get_fields(self):
+        fields = super().get_fields()
+        # This is a hack to allow fields with special characters
+        fields['eo:gsd'] = fields.pop('eo_gsd')
+        return fields
+
+
+class BboxSerializer(gis_serializers.GeoFeatureModelSerializer):
+
+    class Meta:
+        model = Item
+        geo_field = "geometry"
+        auto_bbox = True
+        fields = ['geometry']
+
+    def to_representation(self, instance):
+        python_native = super().to_representation(instance)
+        return python_native['bbox']
+
+
+class AssetsDictSerializer(DictSerializer):
+    # pylint: disable=abstract-method
+    key_identifier = 'asset_name'
+
+
+class AssetsItemSerializer(serializers.ModelSerializer):
```

Comment:
Not sure, if we need to explicitly define the `id` field, since we override the auto-generated primary key. But probably the declarations in `models.py` are enough and no further explicitness needed here.